tests/test_utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+import numpy as np
+from tempfile import TemporaryDirectory
+from autowebcompat import utils
+from PIL import Image
+
+
+def test_get_bugs():
+    bugs = utils.get_bugs()
+    assert(isinstance(bugs, list))
+
+
+def test_mkdir():
+    d = TemporaryDirectory()
+    direc_path = os.path.join(d.name + "\\test")
+    utils.mkdir(direc_path)
+    assert(os.path.isdir(direc_path))
+    utils.mkdir(direc_path)
+
+
+def test_load_image():
+    os.mkdir('data_resized')
+    fake_img = np.random.rand(30, 30, 3) * 255
+    img = Image.fromarray(fake_img.astype('uint8')).convert('RGBA')
```

Comment:
Use `Image.new` instead of creating a numpy array and then convering it into an image.